tests/functions.sh

```diff
@@ -788,6 +788,21 @@ function ts_device_has {
 	return $res
 }
 
+# Based on __SYSMACROS_DEFINE_MAKEDEV macro
+# defined in /usr/include/bits/sysmacros.h of GNU libc.
+function ts_device_make
```

Comment:
What about `ts_makedev` to make it more obvious? :-)

Reply:
Updated.